addon/mixins/focusing.js

```diff
@@ -1,21 +1,28 @@
-import Ember from "ember";
-import getOwner from 'ember-getowner-polyfill';
+import Ember from 'ember';
 
-const { get } = Ember;
+/* globals getOwner */
+// One of these two will be uncommented at build time:
```

Comment:
Hmmm, I'm not sure I like this 😈.  Is the idea to just use "native" `Ember.getOwner` when possible, and avoid including `ember-getowner-polyfill` if it isn't needed?

* We should make ember-getowner-polyfill a "real" polyfill and just provide `Ember.getOwner`, then when it is running with Ember 2.2+ it can literally noop itself (this is roughly what the ember-get-helper addon does to great effect).
* If we are going to keep this logic here (instead of the point above) it should be moved to a specific util module...